main.go

```diff
@@ -70,6 +70,7 @@ func main() {
 	go func() {
 		select {
 		case <-sigChan:
+			fmt.Println("SIGCHAN")
```

Comment:
Is this really meant to be left in prod code? Maybe make it a debug level log?

Reply:
Ohmy. No, this should go away of course. Thanks :)